### ISSUE TITLE
Support building and testing with go1.15

### ISSUE THREAD

**Issue by pmalek-sumo** [influxdata/telegraf#8030]
[go1.15 was released on 11 August 2020](https://blog.golang.org/go1.15) and because [README.md](https://github.com/influxdata/telegraf/blob/6e8255c/README.md) mentions that 

> Telegraf requires Go version 1.13 or newer

the 'contract' isn't fulfilled.

After some testing I have not experienced any problems while building but there is at least 1 test that fails and needs a fix.

### System info:

`go1.15`

### Steps to reproduce:

1. Try to run tests via e.g. `make test`

### Expected behavior:

No failures

### Actual behavior:

`eventhub_consumer` tests break

```
# github.com/influxdata/telegraf/plugins/inputs/eventhub_consumer
plugins/inputs/eventhub_consumer/eventhub_consumer.go:379:40: conversion from int16 to string yields a string of one rune, not a string of digits (did you mean fmt.Sprint(x)?)
```

**Comment by pmalek-sumo** [comment]
https://github.com/influxdata/telegraf/pull/8006 should fix this issue.

**Comment by reimda** [comment]
merged #8006
